vllm/model_executor/models/llama.py

```diff
@@ -491,6 +491,9 @@ class LlamaForCausalLM(nn.Module, SupportsLoRA, SupportsPP):
         "qscale_act": "input_scale",
         "qscale_weight": "weight_scale",
         "kv_fake_quantizer.qscale_act": "kv_scale",
+        "q_fake_quantizer.qscale_act": "attn.q_scale",
```

Comment:
Not super related to this PR, but we the community seems to also want to use some quantization models with q,k,v being applied separately. Thought I could snug this one in here as well.